base/deprecated.jl

```diff
@@ -1632,6 +1632,12 @@ function SymTridiagonal(dv::AbstractVector{T}, ev::AbstractVector{S}) where {T,S
     SymTridiagonal(convert(Vector{R}, dv), convert(Vector{R}, ev))
 end
 
+# issue #22791
+@deprecate_binding select partialsort
```

Comment:
Yeah, I'm not sure what's the difference. Since the function itself is going to be removed (not only some methods), I figured deprecating the binding makes sense. Are there any unintended consequences?